pandas/_typing.py

```diff
@@ -32,7 +32,7 @@
 FilePathOrBuffer = Union[str, Path, IO[AnyStr]]
 
 FrameOrSeries = TypeVar("FrameOrSeries", bound="NDFrame")
-Scalar = Union[str, int, float, bool]
```

Comment:
how is this redundant? try an int is a float, but so it a bool technically, we want to be very explicit on these types.